python/ucxx/_lib_async/application_context.py

```diff
@@ -409,6 +409,12 @@ def get_ucp_worker(self):
         """
         return self.worker.handle
 
+    def get_ucxx_worker(self):
```

Comment:
Nope, this is just the way UCX-Py was written, I ended up replicating the same API to reduce breakage. Although, I agree we could benefit moving those to properties, if anything to make things more Pythonic and consistent. I would prefer to address this on a follow-up PR next week though as I think this will require more effort than immediately noticeable, I opened https://github.com/rapidsai/ucxx/issues/129 to do that. I would also prefer to leave getters here and tackle those too in a follow-up PR if you don't mind.